scripts/deploy

```diff
@@ -63,61 +63,52 @@ namespaces=$(kubectl get namespaces)
 # Automatically create the namespace if it doesn't exist
 echo "Checking if $NAMESPACE namespace exists..."
 if ! echo "$namespaces" | grep -q $NAMESPACE; then
-  kubectl create ns $NAMESPACE
+  kubectl apply -f ../k8s/helium-ns.yaml
```

Comment:
just curious - what's the advantage of applying this vs calling `create` ?

Reply:
Having the namespace be its own file will let us include as part of the helm chart deployment and let us just deploy the whole stack with a helm install command instead of needing to script things to call `kubectl` first